.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Docker",
   // keep in sync with "run_docker"
-  "image": "pyodide/pyodide-env:20250116-chrome132-firefox134",
+  "image": "pyodide/pyodide-env:20250311-chrome134-firefox136-py313",
```

Comment:
We should update `run_docker` as well.

Reply:
It looks like it's already updated in #5498.